workers/tasks/export/html.js

```diff
@@ -75,27 +76,49 @@ const addAttrsToNodes = (newAttrs, matchNodeTypes, nodes) =>
 		return node;
 	});
 
-const addHrefsToNotes = (nodes) =>
-	addAttrsToNodes(
+const getCitationLinkage = (unstructuredValue, structuredValue) => {
+	const digest = digestCitation(unstructuredValue, structuredValue);
+	return {
+		inlineElementId: `citation-${digest}-inline`,
+		bottomElementId: `citation-${digest}-bottom`,
+	};
+};
+
+const getFootnoteLinkage = (index) => {
+	return {
+		inlineElementId: `fn-${index}-inline`,
+		bottomElementId: `fn-${index}-bottom`,
+	};
+};
+
+const addHrefsToNotes = (nodes) => {
```

Comment:
This change is necessary because we don't have access to `count` values at this point in the process. If it turns out that needing to read reactive attributes becomes a more common part of the export process, I will want to rethink this, but for now it seems simpler to manually assign IDs to citations by their value — thus allowing us to link them to the document-bottom notes sections — than to move the "reacting" of the document further up in the pipeline.